doc/src/Tools.rst

```diff
@@ -376,21 +376,36 @@ See README file in the tools/fep directory.
 
 .. _ipi:
 
-i-pi tool
+i-PI tool
 -------------------
 
-The tools/i-pi directory contains a version of the i-PI package, with
-all the LAMMPS-unrelated files removed.  It is provided so that it can
-be used with the :doc:`fix ipi <fix_ipi>` command to perform
-path-integral molecular dynamics (PIMD).
+The tools/i-pi directory used to contain a version of the i-PI package.
+This version, however, was not updated and thus have become outdated.
```

Comment:
I think "was not updated and thus have become outdated" is a little confusing.  May it should say something like:

The tools/i-pi directory used to contain a version of the i-PI package, however, this version was removed and i-PI is now available via PyPi using the pip package manager at ...

Reply:
Done